server/views/account/email-signin.jade

```diff
@@ -10,7 +10,7 @@ block content
                     #flash-content
         .row
           .text-center
-            h2 Sign in or Sign Up with an Email here:
+            h2 Create a new account or Sign in with an Email here:
```

Comment:
The capitalization here is weird. 
Maybe adding some emphasis on `New` and `Sign In` will make it more legible